sdk/storage/azure-storage-blob/README.md

```diff
@@ -156,7 +158,7 @@ The following sections provide several code snippets covering some of the most c
 
 Create a `BlobServiceClient` using the [`sasToken`](#get-credentials) generated above.
 
-<!-- embedme ./src/samples/java/com/azure/storage/blob/ReadmeSamples.java#L30-L33 -->
+<!-- embedme ./src/samples/java/com/azure/storage/blob/ReadmeSamples.java#L31-L34 -->
```

Comment:
@srnagar, do we know if embedme has resolved our issue about supporting named blocks so we don't have a painful experience when changing README sample files?